### ISSUE TITLE
Improving information presented in functions

### ISSUE THREAD

**Issue by RSKothari** [javiribera/locating-objects-without-bboxes#25]
Hi,

Great work. Might be including in future papers of mine. Found one issue:
https://github.com/javiribera/locating-objects-without-bboxes/blob/master/object-locator/losses.py

Might help if you explicitly mentioned that in the case for 1 GT point, be sure to provide it as list of a [1, 2] tensors. This shape is consistent with your code and would lead to failure if people provided GT points as [2, ] or [2, 1].

**Comment by javiribera** [comment]
Yes, both suggestions would make it more convenient for the novice user. Would you do a pull request? That'd be greatly appreciated.
